common/lib/xmodule/xmodule/js/src/video/04_video_full_screen.js

```diff
@@ -1,15 +1,13 @@
 (function (define) {
 'use strict';
-define('video/04_video_full_screen.js', [], function () {
+define('video/04_video_full_screen.js', [], function (HtmlUtils) {
```

Comment:
You need to include the path to HTMLUtils, 'teams/js/views/team_utils'.

Search in LMS JavaScript to see how it should be defined. You can then also run the JS tests locally (they are pretty fast) to make sure it fixed the failures.